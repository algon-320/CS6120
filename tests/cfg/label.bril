@main {
    v1: int = const 1;
.l2:
    v2: int = const 10;
.l3:
.l4:
    v3: int = add v1 v2;
    print v3;
.l5:
}
