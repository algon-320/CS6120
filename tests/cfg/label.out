digraph main {
  node [shape=box];
  bb0;
  l2;
  l3;
  l4;
  l5;
  bb0 -> { l2 }
  l2 -> { l3 }
  l3 -> { l4 }
  l4 -> { l5 }
  l5 -> {  }
}
